openslides_backend/action/action.py

```diff
@@ -397,10 +397,48 @@ def process_write_requests(
                 self.apply_event(event)
                 events_by_type[event["type"]].append(event)
             write_request.events = []
-            for event_type in (EventType.Create, EventType.Update, EventType.Delete):
-                write_request.events.extend(events_by_type[event_type])
+            write_request.events.extend(events_by_type[EventType.Create])
+            write_request.events.extend(
+                self.merge_update_events(events_by_type[EventType.Update])
+            )
+            write_request.events.extend(events_by_type[EventType.Delete])
         return write_request
 
+    def merge_update_events(self, update_events: List[Event]) -> List[Event]:
+        """
+        This is optimation to reduce the amount of update events.
+        """
+        events_by_fqid = defaultdict(list)
+        for event in update_events:
+            events_by_fqid[event["fqid"]].append(event)
+
+        result: List[Event] = []
+        for fqid in events_by_fqid:
+            result.extend(self.merge_update_events_helper(events_by_fqid[fqid]))
+
+        return result
+
+    def merge_update_events_helper(self, events: List[Event]) -> List[Event]:
```

Comment:
```suggestion
    def merge_update_events_for_fqid(self, events: List[Event]) -> List[Event]:
```
helper is always very generic

Reply:
Done. Renamed.